src/components/ProgressBar/ProgressBar.visual.tsx

```diff
@@ -0,0 +1,10 @@
+import * as React from 'react';
+import { visualize, story, snap } from 'storybook-snapper';
+import { ProgressBar } from './';
+
+visualize('ProgressBar', () => {
+  story('simple example', () => {
+    snap('default props', <ProgressBar />);
+    snap('custom buttonText', <ProgressBar buttonText={'Some custom text'} />);
```

Comment:
We should probably change the `buttonText` to `value`

We should probably add some examples with different values (0, 30%, 100%)
and also with and without the text indicator

Reply:
I didnt touch tests at all. Touching it now :)